src/module.c

```diff
@@ -3379,6 +3379,51 @@ int RM_GetClientInfoById(void *ci, uint64_t id) {
     return modulePopulateClientInfoStructure(ci,client,structver);
 }
 
+/* Sets the name of the current client connection, i.e. the client calling the
+ * currently active module command. This is equivalent to the client calling
+ * `CLIENT SETNAME name`.
```

Comment:
```suggestion
/* Sets the name of the client with the given ID. This is equivalent to the client calling
 * `CLIENT SETNAME name`.
```

Reply:
Ah, thanks! I changed the implementation but forgot to change the comment. :D